config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbinstances.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -180,6 +180,29 @@ spec:
                     description: The number of CPU's in the VM instance.
                     type: integer
                 type: object
+              pscInstanceConfig:
+                description: Configuration for Private Service Connect (PSC) for the
+                  instance.
+                properties:
+                  allowedConsumerProjects:
+                    description: |-
+                      List of consumer projects that are allowed to create PSC endpoints to service-attachments to this instance.
+                      These should be specified as project numbers only.
+                    items:
+                      type: string
+                    type: array
+                  pscDnsName:
```

Comment:
How was this DNS created in a real world scenario? This smells like a reference field if the PSC connectivity instance maps to a GCP resource.

Reply:
This also does not point a GCP resource.

Reply:
I double checked the API reference, looks like this is an output-only spec field: https://cloud.google.com/alloydb/docs/reference/rest/v1beta/projects.locations.clusters.instances#PscInstanceConfig

I think we shouldn't keep introducing new output-only spec field because we don't recommend `state-into-spec: merge`, and without it, output-only spec fields won't be populated at all.

I'll need to work on a fix.

Reply:
To unblock this PR, I suggest you ignore pscDnsName and serviceAttachmentLink fields for now. There is no eta for the fix at the moment.

Instructions about how to ignore a field can be found at bullet point 11 in https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#add-the-resource-config-to-service-mappings